app/services/trading_strategies.py

```
@@ -4153,6 +4153,405 @@ def _calculate_portfolio_sortino(self, positions: List[Dict]) -> float:
             self.logger.error(f"Portfolio Sortino calculation failed: {e}")
             return 0.0
 
+    async def _get_symbol_price(self, exchange: str, symbol: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `exchange`

(ARG002)